src/v/config/configuration.cc

```diff
@@ -812,10 +812,20 @@ configuration::configuration()
   , kafka_connections_max_per_ip(
       *this,
       "kafka_connections_max_per_ip",
-      "Maximum number of Kafka client connections from each IP addreess, per "
+      "Maximum number of Kafka client connections from each IP address, per "
       "broker",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       std::nullopt)
+  , kafka_connections_max_overrides(
+      *this,
+      "kafka_connections_max_overrides",
+      "Per-IP overrides of kafka connection count limit, list of "
+      "<ip>:<count> strings",
+      {.needs_restart = needs_restart::no,
+       .example = R"(['127.0.0.1:90', '50.20.1.1:40'])",
```

Comment:
nit: this looks like `<ip_address>:<port>` at first. Maybe `<ip_address>=<limit>` would be better ?

Reply:
@VadimPlh I was doing this symmetrically with the connection rate limiting config, what do you think about changing both of them to use "=" before we release?

Reply:
I did it similar with kafka. As I remember they use ':'

Reply:
I think I'm inclined to stick with `:` here, partly because of kafka, but mainly because it looks more "normal" in a YAML file where that's the usual character for key-value associations.